docs/user-guide/ze-profiles.md

```diff
@@ -120,3 +121,45 @@ You can use this example to customize your own profiles in a configuration file.
   "plugins": []
 }
 ```
+
+## Using base profiles and tokens with existing profiles
+
+As a Zowe user, you can leverage the base profile functionality to access multiple services through Single Sign-on. Base profiles enable you to authenticate using Zowe API Mediation Layer (API ML). You can use base profiles with more than one service profile. For more information, see [Base Profiles](../user-guide/cli-using-using-profiles-v1.md#base-profiles).
+
+Before you log in and connect your service profile, ensure that you have [Zowe CLI](../user-guide/cli-install-cli-checklist.md) v6.16 or higher installed.
+
+### Accessing services through API ML using SSO
+
+Connect your service profile with a base profile and token:
+
+1. Right-click on a profile on one of the ZE trees, the profiles you want to connect through with the api ml
+2. Select Manage Profile from the context menu
+3. In the quick pick, select Log in to Authentication Service
+4. In the next quick pick, select the appropriate option for authenticating to the API ML (user/pw or a cert file)
+5. Enter the into prompted for (credentials, or browsing to the certificate file)
+
+    Tokens are stored in a base profile. If no base profile exists, one is created. A local base profile is created that contains your token. For more information about the process, see [Token Management](../user-guide/cli-using-integrating-apiml.md#how-token-management-works).
+
+:::note
+
+- The feature is only available for base profiles.
+- The feature supports only API Mediation Layer at the moment. Other extenders may use a different authentication service.
+
+:::
+
+### Logging out of the API ML using SSO
+
+If you do not want to store your token, request the server to revoke your token and delete it from your local profile. Use the **Log out from Authentication Service** feature to invalidate the token.
+
+1. Open Zowe Explorer.
+2. Right-click your profile.
+3. Select the **Manage Profile** option.
+4. In the **Quick Pick**, select the **Log out from Authentication Service** option.
+
+    Your token has been successfully invalidated.
+
+### Accessing services for multiple API ML instances
+
+The preceding steps assume you are using a single API ML instance. If you have multiple instances of API ML, the same steps apply, but you must first edit your configuration file to follow a specific structure.
+
+Review [Accessing services for multiple API ML instances](../user-guide/cli-using-creating-profiles.md#accessing-services-for-multiple-api-ml-instances) to see how profiles are structured in the configuration file.
```

Comment:
Would it make sense to turn this section into a note, and move it higher in the article so readers would see it before they follow the log-in/log-out steps?

> [!NOTE]  
> If you want to access services thru multiple API ML gateways, the same steps apply, but you must first edit your configuration file to follow a specific structure.
> Review [Accessing services for multiple API ML instances](../user-guide/cli-using-creating-profiles.md#accessing-services-for-multiple-api-ml-instances) to see how profiles are structured in the configuration file.